genesiscommunitysuccess/blank-app-seed
# fix: address npx lerna using v7 and failing - FUI-1407

[Review Guidance](https://www.notion.so/genesisglobal/Platform-Code-Review-Process-ace93ba760cc4563b0dfb712e2a88d8a)

📓 &nbsp; **Related JIRA**

[FUI-1407](https://genesisglobal.atlassian.net/browse/FUI-1407)

🤔 &nbsp; **What does this PR do?**

- Address "fresh scenario" where `lerna@7` gets installed and will make it "impossible" to build/bootstrap the client folder.
  - Fix to `6.6.2` version
  - Also add `--yes` flag so it doesn't stop/wait for confirmation
- Add `--yes` npx flag on `husky` install (same as item above)

🚀 &nbsp; **Where should the reviewer start?**

Right [here](https://github.com/genesiscommunitysuccess/blank-app-seed/pull/59/files#diff-1846122c2c83a486a3693f7966aa522c34cf489f674185c4da0d9221683fd81fR9).

📑 &nbsp; **How should this be tested?**

- Fresh clone of this repo
- Make sure you don't have npm/npx caches 
- `npx --yes clear-npx-cache`
- Go to the `client` folder
- `npm run bootstrap`

No errors should happen.

On `develop` do the same thing, you'll see this:

![lerna-stop](https://github.com/genesiscommunitysuccess/blank-app-seed/assets/1767830/d8bfeb72-7d0d-4fde-829f-fe980c31e928)

✅ &nbsp; **Checklist**
